src/clusterfuzz/_internal/bot/tasks/utasks/uworker_io.py

```diff
@@ -448,23 +463,29 @@ class UworkerInput(UworkerMsg):
   PROTO_CLS = uworker_msg_pb2.Input
 
   def save_rich_type(self, attribute, value):
-    field = getattr(self.proto, attribute)
+    field = getattr(self._proto, attribute)
     if isinstance(value, dict):
       save_json_field(field, value)
       return
 
     if isinstance(field, collections.abc.Sequence):
       # This the way to tell if it's a repeated field.
       # We can't get the type of the repeated field directly.
+      del field[:]
       value = list(value)
       if len(value) == 0:
         return
-      assert isinstance(value[0], ndb.Model), value[0]
-      field.extend([model._entity_to_protobuf(entity) for entity in value])  # pylint: disable=protected-access
+      if not isinstance(value[0], ndb.Model):
+        field.extend(value)
+      else:
+        # TODO(metzman): This is probably unnecessary and very complext to deal
```

Comment:
Do we hit this case in practice? If not, can we just `raise UnimplementedError()`?